lib/_http_server.js

```diff
@@ -482,8 +483,15 @@ function connectionListener(socket) {
 
       res.detachSocket(socket);
 
-      if (res._last) {
-        socket.destroySoon();
+      if (res._last) socket.destroySoon();
```

Comment:
I found problem with non keep-alive sockets and here is fix for this case. @indutny, finally ```_last``` in it's untouched state is useful to detect non keep-alive sockets and destroy them as it was before this changes. Now all tests completed successfully.